core/pom.xml

```diff
@@ -138,6 +138,7 @@
         <monitoring-console-api.version>2.0.2</monitoring-console-api.version>
         <metainf-services.version>1.11</metainf-services.version>
         <ldapbp.version>1.0</ldapbp.version>
+        <tomlj.version>1.1.1</tomlj.version>
```

Comment:
Does this need to be here? I can't see any core modules depending on this.

Reply:
removed